.github/workflows/maven.yml

```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        jdk: [1.8.x, 11.0.x, 13.0.x]
+    runs-on: ${{matrix.os}}
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Not sure if it'll fix the windows issues you had but try bumping this to `actions/checkout@v2`